crates/api_models/src/enums.rs

```diff
@@ -1,10 +1,10 @@
 use std::str::FromStr;
 
 pub use common_enums::*;
+#[cfg(feature = "dummy_connector")]
```

Comment:
here also

Reply:
@cookieg13 I encountered some merge conflicts due to these changes, so it seems they may all be needed going forward.